src/modules/stakewise/stakewise-daemon/common/nodeset-client.go

```diff
@@ -29,6 +29,24 @@ const (
 	validatorsPath  string = "validators"
 )
 
+// =================
+// === Requests  ===
+// =================
+type ExitMessageDetails struct {
+	Epoch          string `json:"epoch"`
+	ValidatorIndex string `json:"validator_index"`
+}
+
+type ExitMessage struct {
+	Message   ExitMessageDetails `json:"message"`
+	Signature string             `json:"signature"`
+}
+
+type ExitData struct {
+	Pubkey      string      `json:"pubkey"`
+	ExitMessage ExitMessage `json:"exitMessage"`
```

Comment:
We'll have to check with Andy about the mixing of snake_case (`validator_index`) above and camelCase here.

Reply:
I'll iron a lot of these out with him once I test against the API